docs/Design/Classes/Class Declarations.md

```diff
@@ -164,50 +205,56 @@ class Shape extends Measurements<Point> {
     <tbody>
         <tr>
             <th>CSharp</th>
-            <td>`"class "`</td>
-            <td>`" : "`</td>
-            <td>`""`</td>
-            <td>`"\n{"`</td>
-            <td>`"}"`</td>
+            <td><code>"class "</code></td>
+            <td><code>" : "</code></td>
+            <td><code>" : "</code></td>
+            <td><code>""</code></td>
+            <td><code>"\n{"</code></td>
+            <td><code>"}"</code></td>
         </tr>
         <tr>
             <th>Java</th>
-            <td>`"class "`</td>
-            <td>`" extends "`</td>
-            <td>`""`</td>
-            <td>`"{"`</td>
-            <td>`"}"`</td>
+            <td><code>"class "</code></td>
+            <td><code>" extends "</code></td>
+            <td><code>" implements "</code></td>
+            <td><code>""</code></td>
+            <td><code>"{"</code></td>
+            <td><code>"}"</code></td>
         </tr>
         <tr>
             <th>Python</th>
-            <td>`"def "`</td>
-            <td>`"("`</td>
-            <td>`")"`</td>
-            <td>`":"`</td>
-            <td>`""`</td>
+            <td><code>"def "</code></td>
+            <td><code>"("</code></td>
+            <td><code>""</code></td>
+            <td><code>")"</code></td>
+            <td><code>":"</code></td>
+            <td><code>""</code></td>
         </tr>
         <tr>
             <th>Ruby</th>
-            <td>`"class "`</td>
-            <td>`" < "`</td>
-            <td>`""`</td>
-            <td>`""`</td>
-            <td>`"end"`</td>
+            <td><code>"class "</code></td>
+            <td><code>" < "</code></td>
+            <td><code>""</code></td>
+            <td><code>""</code></td>
+            <td><code>""</code></td>
+            <td><code>"end"</code></td>
         </tr>
         <tr>
             <th>TypeScript</th>
-            <td>`"class "`</td>
-            <td>`" extends"`</td>
-            <td>`""`</td>
-            <td>`"{"`</td>
-            <td>`"}"`</td>
+            <td><code>"class "</code></td>
+            <td><code>" extends "</code></td>
+            <td><code>" implements "</code></td>
+            <td><code>""</code></td>
+            <td><code>"{"</code></td>
+            <td><code>"}"</code></td>
         </tr>
     </tbody>
 </table>
 
 ### Errata
 
-* Some languages such as Java and TypeScript do not support multiple inheritance, so GLS will not.
+* Some languages such as Java and TypeScript do not support multiple class inheritance, so GLS will not.
 * Interfaces do not exist in Python, so marking a class as implementing one is currently out of scope.
 * Adding modifiers such as `"extends"` to generics is currently out of scope.
 * Duck-typed languages such as Ruby and Python have no need for generics, so GLS will skip printing generic information in them.
+* Python and Ruby do not support interfaces, so GLS will not print any information related to implementing them.
```

Comment:
It's preferable to say something like "Some languages" for this, since in the future GLS will support other languages that don't support interfaces.